frontend/catalyst/third_party/oqc/src/tests/CMakeLists.txt

```diff
@@ -1,9 +1,3 @@
-cmake_minimum_required(VERSION 3.20)
-
-project(oqc_runtime_tests)
-
-set(CMAKE_CXX_STANDARD 20)
```

Comment:
Is the intention here that this is no longer a standalone project embedded in the frontend source tree, but rather that they're configured by the parent project?

Was this necessary to get python/pybind11 find_package commands to work or just regular cleanup? (Not an issue, just curious)

Reply:
This was not necessary, but more of a cleanup since it avoids having to duplicate certain commands (like the pybind discovery). If I only put it in the higher level cmake file, and someone did actually try to build these tests as a standalone project, it would fail. Given we that don't normally do this nesting, and that I think tests are conceptually dependant on the code they are testing rather than a standalone project, I opted to remove the command.